### ISSUE TITLE
Cannot find some contacts; database error

### ISSUE THREAD

**Issue by paulvt** [nextcloud/social#428]
**Describe the bug**
When I try to search for "gnome@foss.social" (Mastodon) in the app search, it cannot find it. The Nextcloud log contains a database-related error:

```json
{
  "reqId": "7cCvvPHIfpUvVUdmBeBM",
  "level": 2,
  "time": "2019-02-18T19:02:34+00:00",
  "remoteAddr": "<redacted>",
  "user": "<redacted>",
  "app": "no app in context",
  "method": "GET",
  "url": "/apps/social/api/v1/global/account/info?account=gnome@floss.social",
  "message": "500 - {\"status\":-1,\"exception\":\"OCA\\\\Social\\\\Exceptions\\\\InvalidResourceException\",\"message\":\"An exception occurred while executing 'INSERT INTO \\\"oc_social_cache_documents\\\" (\\\"id\\\", \\\"type\\\", \\\"url\\\", \\\"media_type\\\", \\\"mime_type\\\", \\\"error\\\", \\\"local_copy\\\", \\\"public\\\", \\\"creation\\\") VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\\\"https:\\\\\\/\\\\\\/cloud.luon.net\\\\\\/documents\\\\\\/g\\\\\\/ff817e67-6791-4083-9b6d-35290306afa2\\\", \\\"Image\\\", \\\"https:\\\\\\/\\\\\\/cf.mastohost.com\\\\\\/v1\\\\\\/AUTH_91eb37814936490c95da7b85993cc2ff\\\\\\/floss\\\\\\/accounts\\\\\\/avatars\\\\\\/000\\\\\\/014\\\\\\/040\\\\\\/original\\\\\\/0e340be264a47398.png\\\", \\\"image\\\\\\/png\\\", \\\"\\\", 0, \\\"\\\", \\\"0\\\", \\\"2019-02-18 19:02:34\\\"]:\\n\\nSQLSTATE[22001]: String data, right truncated: 7 ERROR:  value too long for type character varying(127)\"}",
  "userAgent": "<redacted>",
  "version": "15.0.4.0"
}

Note that I am using the PostgreSQL database!
``` 

**To Reproduce**
1. Go to the Social app
2. Click in the search field
3. Type "gnome@floss.social"
4. See error in logs

**Client details:**
 - OS: Debian GNU/Linux 9
 - Browse: Firefox
 - Version: 65.0
 - Device: Desktop

<details>
<summary>Server details</summary>

**Social app version:**
0.1.4

**Operating system**:
Debian GNU/Linux 9.8 (Stretch)

**Web server:**
Nginx 1.10.3

**Database:**
PostgreSQL 9.6 ← !!

**PHP version:**

**Nextcloud version:**
15.0.4.0

</details>

**Comment by daita** [comment]
Should be fixed since 0.2.x

**Comment by paulvt** [comment]
Yes, indeed, this is fixed. Thanks :+1:!
